nixos/tests/audit.nix

```diff
@@ -0,0 +1,24 @@
+{
+
+  name = "audit";
+
+  nodes = {
+    machine =
+      { pkgs, ... }:
+      {
+        environment.systemPackages = [ pkgs.audit ];
```

Comment:
this isn't necessary, because we have https://github.com/NixOS/nixpkgs/blob/d5b740f5cf361be926f8f0b60eec6db277576ebd/nixos/modules/security/auditd.nix#L14